tracer/src/Datadog.Tracer.Native/iast/dataflow.cpp

```diff
@@ -322,6 +323,128 @@ void Dataflow::LoadAspects(WCHAR** aspects, int aspectsLength, UINT32 enabledCat
     _loaded = true;
 }
 
+void Dataflow::LoadSecurityControls()
+{
+    auto securityControlsConfig = shared::GetEnvironmentValue(environment::security_controls_configuration);
+    if (securityControlsConfig.size() > 0)
+    {
+        DataflowAspectClass* aspectClass = nullptr;
+
+        trace::Logger::Debug("Dataflow::LoadSecurityControls -> Processing Security Controls Config... ",
+                             securityControlsConfig);
+        auto securityControls = shared::Split(securityControlsConfig, ';');
+        for (auto securityControlLine : securityControls)
+        {
+            auto securityControl = shared::Trim(securityControlLine);
+            if (securityControl.size() == 0 || securityControl[0] == '#')
+            {
+                continue;
+            }
+
+            auto parts = shared::Split(securityControl, ':');
+            if (parts.size() < 5)
+            {
+                trace::Logger::Warn("Dataflow::LoadSecurityControls -> Detected invalid Security Control: ",
+                                    securityControl);
+                continue;
+            }
+
+            int part = -1;
+            SecurityControlType securityControlType = SecurityControlType::Unknown;
+            if ((int) parts.size() > ++part) // Security control kind
```

Comment:
you could `int part = 0` and avoid the `++`-pre-increment

Reply:
Yep, but it's for copy-paste coherence